user_manual/groupware/sync_thunderbird.rst

```diff
@@ -8,29 +8,38 @@ Synchronizing with Thunderbird
 Recommended method
 ------------------
 
+Contacts
+~~~~~~~~
+
 Since Thunderbird 102, the CardDAV protocol is natively supported.
 
 To enable it:
 
-* On the address book view, click the down arrow near "**New Address Book**" and choose "**Add CardDAV Address Book**".
-* In the next window, type your **user name** and **URL of your server**.
-* The next window will ask for your username and password for this account.
-* The previous window will be refreshed and ask you which address books you wish to sync.
-* Choose and then click **Continue**.
+#. On the address book view, click the down arrow near **New Address Book** and choose **Add CardDAV Address Book**.
+#. In the next window, type your **Username** and **Location**.
```

Comment:
Which **Location**? My location (in the office), the server's (in the basement)? The name of the address book (if there are several)?
Also for translations this might be ambiguous, in some languages in might be only apply for physical locations but not for a server's virtual address.

Reply:
sorry, I didn't know this was out of Thunderbird's interface. Yes the small explanation in this case is a good idea.